zcash_client_backend/src/data_api/testing/transparent.rs

```diff
@@ -146,7 +149,8 @@ where
         let balance = summary.account_balances().get(&account.id()).unwrap();
         // TODO: in the future, we will distinguish between available and total
         // balance according to `min_confirmations`
-        assert_eq!(balance.unshielded(), expected);
+        // assert_eq!(balance.unshielded(), expected);
```

Comment:
```suggestion
        #[allow(deprecated)]
        assert_eq!(balance.unshielded(), expected.total());
```

Reply:
the allow directive would be ignored because of the assertion macro so I did this in two separate LoCs so the warning went away